### DISCUSSION TITLE
Does the python-sdk 3.0.0 work for live transcription

### DISCUSSION THREAD

**Discussion by Nikhilesh-B** [deepgram/community#507] [General help] [answered]
Hey all, noticed that the Quickstart documentation for live transcription is outdated for the latest version of the sdk. Was wondering if anyone has tried to get basic transcription functionality up and running with the newest version of the sdk?

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by davidvonthenen** [comment, marked as answer]
All of the documentation should be updated as of last week (the last updates were for playground/console). If there is any documentation that still needs updated, please let us know and provide a link here!
